extension/background/numbers.js

```diff
@@ -0,0 +1,112 @@
+export const numbers = [
+  "1",
+  "2",
+  "3",
+  "4",
+  "5",
+  "6",
+  "7",
+  "8",
+  "9",
+  "10",
+  "11",
+  "12",
+  "13",
+  "14",
+  "15",
+  "16",
+  "17",
+  "18",
+  "19",
+  "20",
+];
+
+export const cardinals = [
+  "one",
+  "two",
+  "three",
+  "four",
+  "five",
+  "six",
+  "seven",
+  "eight",
+  "nine",
+  "ten",
+  "eleven",
+  "twelve",
+  "thirteen",
+  "fourteen",
+  "fifteen",
+  "sixteen",
+  "seventeen",
+  "eighteen",
+  "nineteen",
+  "twenty",
+];
+
+export const ordinals = [
+  "first",
+  "second",
+  "third",
+  "fourth",
+  "fifth",
+  "sixth",
+  "seventh",
+  "eighth",
+  "ninth",
+  "tenth",
+  "eleventh",
+  "twelfth",
+  "thirteenth",
+  "fourteenth",
+  "fifteenth",
+  "sixteenth",
+  "seventeenth",
+  "eighteenth",
+  "nineteenth",
+  "twentieth",
+];
+
+export const numberedOrdinals = [
+  "1st",
+  "2nd",
+  "3rd",
+  "4th",
+  "5th",
+  "6th",
+  "7th",
+  "8th",
+  "9th",
+  "10th",
+  "11th",
+  "12th",
+  "13th",
+  "14th",
+  "15th",
+  "16th",
+  "17th",
+  "18th",
+  "19th",
+  "20th",
+];
+
+export function findMatchingIndex(itemName) {
+  //Return the index of the itemName from the any of the number arrays that contains it
+  const val = itemName.toLowerCase();
+  const numberIndex = numbers.indexOf(val);
+  const ordinalIndex = ordinals.indexOf(val);
+  const cardinalIndex = cardinals.indexOf(val);
+  const numberedOrdinalsIndex = numberedOrdinals.indexOf(val);
```

Comment:
Are these correct? It seems like "first" is going to return 0, "second" 1 and so on.

Though it's a little more tedious, `"1st": 1, "first": 1, "one": 1, "2nd": 2, ...` might be a better way to define these. Then we could add something like `"1th": 1` later if we somehow see people accidentally saying that (i.e., not assume there's only one way to say each kind of number).